website/project/model.py

```diff
@@ -156,6 +156,34 @@ class MetaData(GuidStoredObject):
     date_modified = fields.DateTimeField(auto_now=datetime.datetime.utcnow)
 
 
+def validate_contributor(guid, contributors):
+    if guid:
+        user = User.find(
+            Q('_id', 'eq', guid) &
+            Q('is_claimed', 'eq', True)
+        )
+        if user.count() != 1:
+            raise ValidationValueError('User does not exist or is not active.')
+        elif guid not in contributors:
+            raise ValidationValueError('Mentioned user is not a contributor.')
+    return True
+
+def get_valid_mentioned_users_guids(comment, contributors):
+    """ Returns a list of valid users mentioned in the comment content.
+
+    :param Node comment: Node that has new_mentions and ever_mentioned
+    :param list new_mentions: List of guids of mentioned users
+    :param list contributors: List of contributors on the node
+    :return bool new_mentions_added: Whether there are valid new_mentions
```

Comment:
We are returning a list, not a boolean, and an inspection of the [call](https://github.com/CenterForOpenScience/osf.io/blob/0c51337fa1034dbeac751a98da63b3bc8492fc06/website/project/model.py#L332) [sites](https://github.com/CenterForOpenScience/osf.io/blob/0c51337fa1034dbeac751a98da63b3bc8492fc06/website/project/model.py#L357) for this function suggests that list is the correct type to return.